packages/malloy/src/model/malloy_query.ts

```diff
@@ -1406,6 +1400,66 @@ class QueryField extends QueryNode {
     }
   }
 
+  generateAppliedFilter(
+    context: QueryStruct,
+    filterMatchExpr: FilterMatchExpr
+  ): string {
+    let filterExpr = filterMatchExpr.kids.filterExpr;
+    while (filterExpr.node === '()') {
+      filterExpr = filterExpr.e;
+    }
+    if (filterExpr.node === 'parameter') {
+      const name = filterExpr.path[0];
```

Comment:
This is also one of the uncomfortable places, happy to get correction here.

Reply:
I think this is correct given the way namespaces work today.... when we fix namespaces this will probably change.